plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -681,10 +681,10 @@ codemodernizer.toolwindow.table.header.status=Status
 codemodernizer.toolwindow.transformation.history.header=Transformation job history
 codemodernizer.toolwindow.transformation.progress.header=Transformation progress
 codemodernizer.toolwindow.transformation.progress.running_time=Running time: {0}
-codewhisperer.actions.connect_github.title=Connect with us on GitHub
+codewhisperer.actions.connect_github.title=Connect with Us on GitHub
```

Comment:
u should probably not be capitalized here.

Reply:
it will give a warning if I don't do so,